### PR TITLE
 feat(design-land): added article component and examples to design-land

### PR DESCRIPTION
**Pull request by Nolan-Arnold**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Fixes: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

## Other information

### CODE REVIEW

File: apps/design-land/src/app/core/code-preview/component/code-preview.component.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+:host {
+	border: 1px solid gray;
```
Comment (xelaint) [inline apps/design-land/src/app/core/code-preview/component/code-preview.component.scss:2]: color needs to use the daffodil palette variables and separated into code preview's own theme file.

File: apps/design-land/src/app/core/code-preview/component/code-preview.module.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { CodePreviewComponent } from './code-preview.component';
+import { HighlightModule } from 'ngx-highlightjs';
+import { DaffNavbarModule } from 'libs/design/src';
```
Comment (xelaint) [inline apps/design-land/src/app/core/code-preview/component/code-preview.module.ts:5]: Should import from `@daffodil/design`

File: apps/design-land/src/app/core/code-preview/component/code-preview.component.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<daff-navbar color="primary" *ngIf="title">
```
Comment (xelaint) [inline apps/design-land/src/app/core/code-preview/component/code-preview.component.html:1]: What's the purpose of the navbar? I'm not seeing it rendered on the page.

**Comment by damienwebdev** [comment]
closing in favor of #892 #893 and #894

**Pull request closed without merging**